PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -28,6 +28,7 @@
 #include "Common/DataModel/PIDResponse.h"
 #include "PWGLF/DataModel/LFStrangenessTables.h"
 #include "Common/DataModel/TrackSelectionTables.h"
+#include "PWGHF/HFC/DataModel/DMesonPairsTables.h"
```

Comment:
I would like to avoid including data models from subfolders in the data model. In fact, I think it is better to move all correlation table declaration into a separate HFC data model. What do you think @fcolamar?